### ISSUE TITLE
A11y

### ISSUE THREAD

**Issue by jaysonmc** [gcdevops/HRWhiteListing#20]

**Comment by gabrielcossette** [comment]
Accessibility doesn't seem a priority for Odoo for now: https://github.com/odoo/odoo/issues/13950

**Comment by jaysonmc** [comment]
Hmm, that's too bad. It's our quick fix any who. Long term we cannot use a non-a11y solution.

**Comment by shawnthompson** [comment]
I'm pretty sure this application falls in the needs to be accessible now. I'm here to help, let me know if you need anything.

**Comment by FredrikArvas** [comment]
Hi
We would like to join forces to solve A11y-issue. 
see https://github.com/odoo/odoo/issues/13950
